src/unstract/sdk/utils/tool_utils.py

```diff
@@ -100,3 +100,20 @@ def get_file_mime_type(input_file: Path) -> str:
             input_file_mime = magic.from_buffer(sample_contents, mime=True)
             input_file_obj.seek(0)
         return input_file_mime
+
+    @staticmethod
+    def get_file_size(input_file: Path) -> int:
+        """Gets the file size in bytes for an input file.
+        Args:
+            input_file (Path): Path object of the input file
+
+        Returns:
+            str: MIME type of the file
+        """
+        with open(input_file, mode="rb") as input_file_obj:
```

Comment:
Should we handle basic errors like file not found here @johnyrahul ?

Reply:
@gaya3-zipstack The calling function is encapsulated with try catch.
Calling the process from the index function.